guides/GITHUB.md

```diff
@@ -0,0 +1,334 @@
+# Getting Started with GitHub
+
+What is GitHub, and how do you use it to interact with and contribute to SourceCred?
+This guide will give you an introduction to GitHub and SourceCred.
+
+## Why do we use GitHub?
+
+For now, think of GitHub as a place to collaborate. 
+If you had a bunch of documents that you wanted to collaborate on, what would you do?
+You might consider putting them in a shared file service like [Google Docs](https://docs.google.com/), or perhaps keeping them as Word documents on your local machine. 
+Although these platforms offer general collaborative editing capabilities, GitHub
+offers a bit more:
+
+### Permissionless collaboration
+
+Anyone can propose changes. You don't need to be given permission to view or edit
+a document, as is required by some of the platforms mentioned previously.
+
+### More advanced reviewing tools
+
+When you open up a Google Doc, you might be able to comment or write in revision mode, but
+when you do so, you are working with the entire document. GitHub allows
+review on the level of change. Along with this more detailed ability to edit,
```

Comment:
Not sure what 'GitHub allows review on the level of change' means.

Reply:
I think I was trying to get at "You can check out a new branch, and instead of globally editing the whole thing, work on your change separately from the individual document." It's definitely not clear! Do you have a suggestion for how it could be said?

Reply:
Maybe something like:
> When you open up a Google Doc, you might be able to comment or write in revision mode, but
when you do so, you are working with the entire document. GitHub allows
you to separate your working changes from the production version, ensuring that review and discussion
can proceed before anything is made final. By way of branches, several people can do this at once
without it being confusing. Along with this more detailed ability to edit,
there is an integrated approval workflow, straightforward visualizations of what changed 
(these are called "diffs"), and powerful automatic testing (called continuous integration or CI).
 
?

Reply:
This is perfect. Thanks!

Reply:
This is perfect! Thanks!